internal/mode/static/nginx/config/maps.go

```diff
@@ -21,6 +22,48 @@ func executeMaps(conf dataplane.Configuration) []executeResult {
 	return []executeResult{result}
 }
 
+func executeStreamMaps(conf dataplane.Configuration) []executeResult {
+	maps := createStreamMaps(conf)
+
+	result := executeResult{
+		dest: streamConfigFile,
+		data: helpers.MustExecuteTemplate(mapsTemplate, maps),
+	}
+
+	return []executeResult{result}
+}
+
+func createStreamMaps(conf dataplane.Configuration) []*http.Map {
```

Comment:
I used a slice of map pointers because I wanted to edit the existing Map when I loop through and I find a server with the same port as an existing map, so then I want to add a new parameter to the existing map. I tried using a slice without pointers at first and it was not updating the maps so I switched to pointers and it worked.

Reply:
You can still edit the element of the slice without pointers, you just have to be careful when editing the element in a range loop: https://goplay.tools/snippet/cZSoIr2cwpt